diff --git a/includes/libs/rdbms/database/DatabaseMySQL.php b/includes/libs/rdbms/database/DatabaseMySQL.php
index e7f0ac5..17aa5ce 100644
--- a/includes/libs/rdbms/database/DatabaseMySQL.php
+++ b/includes/libs/rdbms/database/DatabaseMySQL.php
@@ -384,7 +384,7 @@
 		$res = $this->query( $query, __METHOD__ );
 		$row = $res->fetchObject();
 
-		return $row && (bool)$row->Value;
+		return $row && $row->Value && $row->Value !== 'OFF';
 	}
 
 	/**
